tests/integration/init/schemas/schemas_test_data_setup.py

```diff
@@ -4,7 +4,7 @@
 import tempfile
 from unittest import TestCase
 
-from boto3 import Session
+from boto3.session import Session
```

Comment:
What was the issue? Are the `Session`s identical?

Reply:
it is, mypy is not happy if we import `Session` from `boto3/__init__.py` which imports from `boto3.session`. I don't know how it works, but it seems when file A import X from B, mypy will prevent us from import X from A.